content/dojos/68-robots.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "#68 robots"
+date: 2020-09-05T0:0:0+01:00
+draft: false
+headercolor: "orange-background"
+wanneer: 
+  van: 2020-09-05T12:00:00.0Z
+  tot: 2020-09-05T14:30:00.0Z
+waar: "De Smeltkroes, Waalbandijk 16, 6541 AJ Nijmegen"
+eventbrite:
+  inschrijventot: 2020-09-05T00:00:00.0Z
+  url: "https://www.eventbrite.nl/e/tickets-coderdojo-nijmegen-68-robots-118057602467"
+---
+![Robots](https://img.evbuc.com/https%3A%2F%2Fcdn.evbuc.com%2Fimages%2F109404475%2F187233351803%2F1%2Foriginal.20200824-202316?w=1080&auto=format%2Ccompress&q=75&sharp=10&rect=0%2C0%2C2160%2C1080&s=4924dfd07b2c9f175a6c2803892a4e69)
+## Wat gaan we doen?
+Deze maand mogen de robots hun kunsten weer vertonen. We gaan in ieder geval aan de slag met de micro:Maqueen, een veelzijdig robotje met een micro:bit als hersenen. Hopelijk is ook de mBot weer van de partij. Beide robots zijn te programmeren met een Scratch-achtige taal en met "echte" code - voor elk wat wils dus. 
+
+ <!--more-->
+
+## Voorbereiding
+Neem een laptop mee, en installeer om snel aan de slag te kunnen de [micro:bit offline app](https://makecode.microbit.org/offline-app) en het [mBlock programma](https://mblock.makeblock.com/en-us/download/) op.
```

Comment:
Zouden we hier ook het Corona stukje (zoals op EventBrite) nog bij moeten zetten?

Reply:
Toegevoegd.